src/test/scala/tech/sourced/siva/test/SivaReaderSpec.scala

```diff
@@ -113,12 +96,33 @@ class SivaReaderSpec extends FlatSpec with Matchers {
     sivaReader.close()
   }
 
-  private def getReader(filename: String): SivaReader = {
+}
+
+object SivaReaderSpec {
```

Comment:
Should this object be in a common place for be used for all the specs?

Reply:
Please, feel free to refactor it later. 

This PR includes minimal changes to make the code reusable for cases of Reader and Unpacker test.